src/main/java/io/debezium/connector/yugabytedb/YugabyteDBStreamingChangeEventSource.java

```diff
@@ -484,8 +489,11 @@ else if (message.isDDLMessage()) {
                     // has succeeded
                     retryCount = 0;
                 }
-            }
-            catch (Exception e) {
+            } catch (CDCErrorException cdcException) {
+                // This will cause the connector to restart
+                LOGGER.info("Encountered CDCErrorException, possibly indicating a tablet split", cdcException);
```

Comment:
Why guess? Is it possible to have a specific exception for tablet split?

Reply:
Removed this log line, added it for testing before.